reyna-test/src/com/b2msolutions/reyna/DispatcherTest.java

```diff
@@ -698,4 +698,16 @@ public void whenOldConfigurationIsFromZeroAndToZeroAllowSending() {
 
         assertEquals(Result.OK, Dispatcher.canSend(context));
     }
+
+    @Test
+    public void whenCallingCanSendAndCurrentTimeIsBetweenNonRecurringWwanBlackoutStartTimeAndEndTimeShouldBlackout() {
```

Comment:
Can we add a test for time error where device register time was in the future?

preferences.saveNonRecurringWwanBlackout(70L, 84L);

        GregorianCalendar now = mock(GregorianCalendar.class);
       when(now.getTimeInMillis()).thenReturn(63L);

It should return OK